server/app/boards.go

```diff
@@ -159,14 +182,35 @@ func (a *App) DeleteBoardMember(boardID, userID string) error {
 		return bErr
 	}
 
-	_, err := a.store.GetMemberForBoard(boardID, userID)
+	oldMember, err := a.store.GetMemberForBoard(boardID, userID)
 	if errors.Is(err, sql.ErrNoRows) {
 		return nil
 	}
 	if err != nil {
 		return err
 	}
 
+	// if we're removing an admin, we need to check that there is at
+	// least still another admin on the board
+	if oldMember.SchemeAdmin {
```

Comment:
This code block is a duplicate from `UpdateBoardMember`. Do you think it makes sense to move this to a utility?